paddle/fluid/framework/ir/mkldnn/softplus_activation_mkldnn_fuse_pass.h

```diff
@@ -36,7 +36,7 @@ class SoftplusActivationOneDNNPass : public FusePassBase {
 
   void FuseSoftplusActivation(
       ir::Graph *graph,
-      const std::string &fuse_activation_type,
```

Comment:
Since its only a variable name you can postpone this change to another PR to reduce number of modified files to 20(no approval needed).